server/conn.go

```diff
@@ -116,6 +116,8 @@ func (c *conn) handleRequest(req *pdpb.Request) (*pdpb.Response, error) {
 		return c.handleGetMeta(req)
 	case pdpb.CommandType_PutMeta:
 		return c.handlePutMeta(req)
+	case pdpb.CommandType_AskChangePeer:
+		return c.handleAskChangePeer(req)
```

Comment:
No, here is for Handle ask, and the worker will get the job and do the really request later.